root/layout/components/NewFooter.js

```diff
@@ -0,0 +1,242 @@
+/*
+ * @flow strict
+ * Copyright (C) 2025 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import {faAngleRight} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import * as React from 'react';
+
+import metabrainzLogo
+  from '../../static/images/meb-logos/metabrainz.svg';
+import musicbrainzLogo
+  from '../../static/images/meb-logos/musicbrainz.svg';
+import {l} from '../../static/scripts/common/i18n.js';
+
+component NewFooter() {
+  return (
+    <section className="footer layout-width">
+      <div className="container-fluid px-4">
+        <div className="row align-items-baseline pb-3">
+          <div className="col-12 col-lg-4">
+            <h3>
+              <img
+                alt="MusicBrainz"
+                src={musicbrainzLogo}
+                width="180"
+              />
+            </h3>
+            <br />
+            <p>
+              {l(
+                `is an open music encyclopedia that collects music metadata
+                and makes it available to the public.`,
+              )}
+            </p>
+            <ul className="list-unstyled">
+              <li className="color-a">
+                <span>{l('Development IRC:')} </span>
```

Comment:
Should probably be changed by now:
```suggestion
                <span>{l('Development chat:')} </span>
```

Or maybe even
```suggestion
                <span>{addColonText(l('Development chat'))} </span>
```